### PR TITLE
Create teckin_sb50.yaml

### PR DESCRIPTION
**Pull request by gleepwurp**
Device configuration for Teckin SB-50 RGBW light bulb.

## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>
**Pull request in [esphome-core](https://github.com/esphome/esphome-core) with C++ framework changes (if applicable):** esphome/esphome-core#<esphome-core PR number goes here>

## Checklist:

  - [ ] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.

### CODE REVIEW

File: devices/teckin_sb50.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  name: '<NODE_NAME>'
+  platform: ESP8266
+  board: esp01_1m
+
+wifi:
+  ssid: '<WIFI_SSID>'
+  password: '<WIFI_PASSWORD>'
+
+logger:
+  # Disable UART logging (pins GPIO1/3 are used for button/led)
```
Comment (OttoWinter) [inline devices/teckin_sb50.yaml:11]: I don't see those being used in this config?

File: devices/teckin_sb50.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+esphome:
+  name: '<NODE_NAME>'
+  platform: ESP8266
+  board: esp01_1m
```
Comment (OttoWinter) [inline devices/teckin_sb50.yaml:4]: Does this also need an `on_boot` automation that turns the light on when power is initially applied as in the other LED bulb examples in the devices section?

**Review by OttoWinter — APPROVED** [review]
- Please link to this file in the generic sonoff page. Otherwise no one will find this file.

File: devices/teckin_sb50.yaml (hunk 2 above)
Comment (gleepwurp) [reply devices/teckin_sb50.yaml:4]: hmm, good question.  I'll check and make adjustments before linking, if required.

**Comment by OttoWinter** [comment]
Closing due to inactivity.

**Pull request closed without merging**